File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_empty(int space);
void print_row(int bricks);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int i = 0, a = height; i < height && a > 0; i++, a--)
    {
        print_empty(a - 1);
        print_row(i + 1);
        printf("\n");
    }
}

void print_row(int bricks)
{
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
}

void print_empty(int space)
{
    for (int a = space; a > 0; a--)
    {
        printf(" ");
    }
}

File: sentimental-mario-more/mario.py
height = input("Height: ")
while not height.isnumeric() or int(height) < 1 or int(height) > 8:
    height = input("Height: ")

a = int(height)


def space(space):
    for _ in range(space):
        print(" ", end="")


def row(bricks):
    for _ in range(bricks):
        print("#", end="")


for i in range(a):
    space(a - 1 - i)
    row(i + 1)
    print("  ", end="")
    row(i + 1)
    print()
